verify.sh

```diff
@@ -16,61 +16,61 @@
 #
 # Copyright 2023 The Kepler Contributors
 #
-set -x
+set -eu -o pipefail
 
-NAMESPACE=${NAMESPACE-"monitoring"}
+PROJECT_ROOT="$(git rev-parse --show-toplevel)"
+declare -r PROJECT_ROOT
+source "$PROJECT_ROOT/lib/utils.sh"
+
+declare -r NAMESPACE=${NAMESPACE-"monitoring"}
 
 function rollout_status() {
-    kubectl rollout status $1 --namespace $2 --timeout=5m || {
-        echo "fail to check status of ${1} inside namespace ${2}"
-        exit 1
-    }
+	kubectl rollout status $1 --namespace $2 --timeout=5m || {
+		die "fail to check status of ${1} inside namespace ${2}"
+	}
 }
 
 function verify_bcc() {
-    # basic check for bcc
-    if [ $(dpkg -l | grep bcc | wc -l) == 0 ]; then
-        echo "no bcc package found"
-        exit 1
-    fi
+	# basic check for bcc
+	dpkg -l | grep bcc || die "no bcc binary found"
```

Comment:
can we instead look for a specific binaries instead?

Reply:
we have to look for bcc as bcc is kepler dependence. @sunya-ch , even if we merge ebpf related PR in recently right?